src/systems/filecoin_vm/message/_index.md

```diff
@@ -4,6 +4,47 @@ statusIcon: 🔁
 title: VM Message - Actor Method Invocation
 ---
 
+A message is the unit of communication between two actors, and thus the primitive cause of changes
+in state. A message combines:
+
+- a token amount to be transferred to be transferred from the sender to the receiver, and
+- a method with parameters to be invoked on the receiver (optional).
+
+Actor code may send additional messages to other actors while processing a received message. 
+Messages are processed synchronously: an actor waits for a sent message to complete before resuming control.
+
+The processing of a message consumes units of computation and storage denominated in gas. 
+A message's gas limit provides an upper bound on its computation. The sender of a message pays 
+for the gas units consumed by a message's execution (including all nested messages) at a
+gas price they determine.
+
+# Message validation
+
+A syntactically invalid message should not be transmitted, retained in the message pool, or
+included in a block.
+
+A syntactically valid message:
+
+- has a well-formed, non-empty `To` address ,
+- has a well-formed, non-empty `From` address, 
+- has distinct `To` and `From ` addresses,
```

Comment:
may be a dumb question but is that truly necessary? Could an actor never call a method on itself to update its own state?